docs/source/blueprints/BP005.rst

```diff
@@ -0,0 +1,295 @@
+
+4. Reduce traffic in the main framework repository and allow for a scalable
+   community on its own standalone project
+
+   As evidenced by “avocado-misc-tests”, a more focused standalone project will
+   attract a larger number of similar minded developers and users. By being a
+   simpler standalone repository, with its own specific contributions
+   guidelines, the entry barriers are also lowered.
+
+
+Specification
+*************
+
+To avoid the need to change our current imports, our first attempt was to keep
+the "avocado.utils" namespace regardless of the location of its codebase by
+using the Implicit Package Namespace (PEP-420) approach. Namespace packages are
+a mechanism for splitting a single Python package across multiple directories
+on disk. Unfortunately, to be able to convert the avocado.utils.* modules and
+packages into a namespace package, we would need to remove the `__init__.py`
+files from multiple places. That would break, among other things, the current
+Avocado egg based deployment model used on spawners such as podman.
+
+Because of that, we decided to have a new namespace for the utility libraries.
+We ran a poll to choose the name and `autils` (Avocado Utils) was the winner.
+So our goal is to host common libraries under `avocado-project/autils`
+repository on GitHub and make them also available on PyPI under `python-autils`
+package name, since the `autils` name is unavailable.
+
+Utility onboarding
+==================
+
+The primary goal of autils is to be a central place for modules that ease the
+interaction with system level features and interfaces.  As such, if a user
+needs to interact with such a system level feature or interface, he/she may
+propose a new module.
+
+For instance, if a user finds himself/herself constantly interacting with disk
+partitions, he/she may want to propose a “partitions” module that provides
+basic APIs for that. In this case it makes sense to have it as part of this
+central repository.
+
+However, if the utility intends to ease the interaction with an esoteric
+feature, system or device, this may not be a good place for it.
+
+As a general rule, if a system level feature or interface is generically
+available to a multitude of users, such as the partition example above, it is a
+good candidate for this repository. Common sense is much appreciated here and
+we can always review the proposals during the PR submission or within a Github
+discussion thread (even before the proposal is ready for submission).
+
+Besides that, the proposed utility must have at least one MAINTAINER that will
+be responsible for keeping track of Pull Requests, tests and quality
+requirements as described below:
+
+Quality requirements
+--------------------
+
+It’s expected that both new modules and changes to existing modules will need
+to observe a full set of quality requirements and general standards that will
+either be described (documentation) and enforced as automated checks as much as
```

Comment:
The "either" word here doesn't work well with the "and ...".  I'd suggest:

"... that will be described in documentation and enforced as automated checks as much as ..."